src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -702,6 +702,7 @@ export class CalciteCombobox {
       return (
         <calcite-chip
           class={chipClasses}
+          data-item={item}
```

Comment:
Another way to do this would be to not use `@Listen` and just add a handler to the chips themselves:

```tsx
<calcite-chip
          class={chipClasses}
    onCalciteChipDismiss={() => {
        this.active = false;
        this.toggleSelection(item, false);
    })
  />
```

That way you don't have to look up anything, as you already have the item.

Reply:
@paulcpederson @jcfranco if I do the above, I will get a linting error to not use arrow functions in JSX. Using bind isn't allowed with linter either.

- JSX props should not use arrow functionseslintreact/jsx-no-bind
- JSX props should not use .bind()eslintreact/jsx-no-bind

What would be the recommended approach here?

Reply:
that seems pretty restrictive...

Reply:
You should be getting a warning, not an error. I avoid anonymous functions whenever possible, but it's not enforced at the moment.

Reply:
Sorry, yes it's a warning but I am trying to follow the warning :)